charmcraft/templates/init-go-framework/.gitignore.j2

```diff
@@ -0,0 +1,9 @@
+venv/
+build/
+*.charm
+.tox/
```

Comment:
Perhaps this isn't needed?

Reply:
After reviewing, I'd rather leave it there, in case there it a tox file for things like tests or linting.